### PR TITLE
Make code readable

### PR DESCRIPTION
**Pull request by konstantinosmpouros**
Creating the arguments adding in a separate method with docstrings

### CODE REVIEW

**Comment by DanielAzulayy** [comment]
That's a pretty good idea, the whole project needs refactoring, there are some classes and functions that can easily be broken into subfunctions like what you did with `check_both_output_and_folderoutput`.

**Comment by konstantinosmpouros** [comment]
@DanielAzulayy that's exactly what l am trying to do cause with function we can easily see each step of the process and make easier the maintenance as well. l am working more for now in the main of sherlock.py

File: sherlock/sherlock.py [hunk 1]
```diff
@@ -600,17 +622,55 @@ def main():
...
+def sites_path(args):
+    """
+        Filling the site variable with the information of the sites we will search.
+        if the user uses the argument local sherlock will check the data.json file that he can store
+        new sites to check.
+    """
     try:
         if args.local:
-            sites = SitesInformation(os.path.join(
+            sites = SitesInformation(o.path.join(
```
Comment (JassonCordones) [inline sherlock/sherlock.py:634]: typo in the os

File: sherlock/sherlock.py [hunk 2]
```diff
@@ -474,14 +474,35 @@ def timeout_check(value):
...
+            print("Update Available!\n" +
+                  f"You are running version {local_version}. Version {remote_version} is available at https://git.io/sherlock")
+
+    except Exception as error:
+        print(f"A problem occurred while checking for an update: {error}")
+
+
+def adding_arguments(parser):
+    """
+    In this method arguments that is used in the cmd, for example --version are
```
Comment (JassonCordones) [inline sherlock/sherlock.py:499]: In this method arguments that **_are_** used in the cmd

File: sherlock/sherlock.py [hunk 3]
```diff
@@ -474,14 +474,35 @@ def timeout_check(value):
...
+
+        if remote_version != local_version:
+            print("Update Available!\n" +
+                  f"You are running version {local_version}. Version {remote_version} is available at https://git.io/sherlock")
+
+    except Exception as error:
+        print(f"A problem occurred while checking for an update: {error}")
+
+
+def adding_arguments(parser):
```
Comment (JassonCordones) [inline sherlock/sherlock.py:497]: The name of this function is not really clear about what it does. The verb shouldn't be in infinitive, **_add_arguments_** would be a better naming convention

File: sherlock/sherlock.py [hunk 4]
```diff
@@ -600,17 +622,55 @@ def main():
         print("You can only use --output with a single username")
         sys.exit(1)
 
-    # Create object with all information about sites we are aware of.
+
+def sites_path(args):
+    """
+        Filling the site variable with the information of the sites we will search.
```
Comment (JassonCordones) [inline sherlock/sherlock.py:628]: **Fills** the site variable with the information of the sites we will search

**Pull request closed without merging**